tests/routes/get_favorite_test.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from models import PcObject
+from tests.conftest import clean_database, TestClient
+from tests.test_utils import API_URL, create_user, create_offerer, create_venue, create_offer_with_thing_product, \
+    create_mediation, create_favorite, create_booking, create_stock
+from utils.human_ids import humanize
+
+
+@pytest.mark.standalone
+class Get:
+    class Returns200:
+        @clean_database
+        def when_user_is_logged_in_and_a_favorite_exist(self, app):
+            # Given
+            user = create_user()
+            offerer = create_offerer()
+            venue = create_venue(offerer, postal_code='29100', siret='12345678912341')
+            offer = create_offer_with_thing_product(venue, thumb_count=0)
+            mediation = create_mediation(offer, is_active=True)
+            favorite = create_favorite(mediation, offer, user)
+            PcObject.save(user, favorite)
+
+            # When
+            response = TestClient(app.test_client()).with_auth(user.email) \
+                .get(API_URL + f'/favorites/{humanize(favorite.id)}')
```

Comment:
```suggestion
                .get(f'{API_URL}/favorites/{humanize(favorite.id)}')
```
Mets ça dans une variable `url`.

Reply:
:+1: